app/views/coronavirus_form/check_answers.html.erb

```diff
@@ -13,20 +13,16 @@
   margin_bottom: 3,
 } %>
 
-<%= tag.p t("check_your_answers.description"), class: "govuk-body" %>
+<div class="govuk-!-margin-bottom-8">
+  <% sections.each do |section| %>
+    <%= render "coronavirus_form/check_your_answers/sections/#{section}" %>
+  <% end %>
+</div>
 
-<%= render "govuk_publishing_components/components/summary_list", {
-  items: items,
+<%= render "govuk_publishing_components/components/inset_text", {
+  text: t("check_your_answers.confirmation")
 } %>
```

Comment:
Really picky, but the curly braces are not needed. So:
```ruby
render "govuk_publishing_components/components/inset_text", text: t("check_your_answers.confirmation")
```

Reply:
Hi @reggieb, sorry you are incorrect - the braces are required.  The page throws a syntax error without them.